src/client.cpp

```diff
@@ -518,6 +518,9 @@ void Client::step(float dtime)
 
 			// Send as unreliable
 			Send(1, data, false);
+
+			// Empty the password, don't let it in memory !!!
+			m_password = "";
```

Comment:
If I'm not completely mistaken, this does not guarantee that the existing memory is overwritten in any way.